canu/2.1.1/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Use ubuntu as base image
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
```

Comment:
please adjust line 6 to: `LABEL dockerfile.version="1"`

see explanation here https://github.com/StaPH-B/docker-builds/issues/118

Reply:
gah. I see it below. You can just delete this LABEL

Reply:
Fixed by f87b8b643f33178a43053fa0e661140d0f7ce0bf